class.jetpack.php

```diff
@@ -5792,7 +5792,7 @@ public static function xmlrpc_async_call( ...$args ) {
 		$client_blog_id = is_multisite() ? $blog_id : 0;
 
 		if ( ! isset( $clients[ $client_blog_id ] ) ) {
-			$clients[ $client_blog_id ] = new Jetpack_IXR_ClientMulticall( array( 'user_id' => JETPACK_MASTER_USER ) );
+			$clients[ $client_blog_id ] = new Jetpack_IXR_ClientMulticall( array( 'user_id' => JETPACK_MASTER_USER ) ); //phpcs:ignore ..Jetpack_Sniffs_MasterUserConstant.ShouldNotBeUsed
```

Comment:
Since we are not getting rid of `JETPACK_MASTER_USER` here, should we replace this with `Manager::MASTER_USER`?
Or may be leave a comment about why the deprecated feature is used, so somebody could fix it at some point?

Reply:
@sergeymitr finally got back here and finished it!